Server/API/OrganizationManagementController.cs

```diff
@@ -89,6 +91,20 @@ public async Task<IActionResult> DeleteUser(string userID)
             return Ok("ok");
         }
 
+        [HttpGet("DeviceGroup")]
+        [ServiceFilter(typeof(ApiAuthorizationFilter))]
+        public IActionResult DeviceGroup()
+        {
+            if (User.Identity.IsAuthenticated &&
+                DataService.GetUserByNameWithOrg(User.Identity.Name).IsAdministrator)
+            {
+                return Ok(DataService.GetDeviceGroups(User.Identity.Name));
+            }
+            if (Request.Headers.TryGetValue("OrganizationID", out var orgID))
+                return Ok(DataService.GetDeviceGroupsForOrganization(orgID));
+            return BadRequest("Unable to find User or organizationID for device group");
```

Comment:
Another one of those pedantic things where I'd return `NotFound`.  :D

Reply:
@stevesobol Can I get you to make this change?

Reply:
Done